src/components/SongHistory.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+
+class SongHistory extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      displayList: false,
+    }
+  }
+
+  toggleDisplay = () => {
+    this.setState({
+      displayList: !this.state.displayList
+    })
+  }
+
+  render(){
+    const {songHistory} = this.props;
+    const songs = songHistory.map(song => song.song).reverse()
+    return(
+      <div id='recent-song-history' onClick={this.toggleDisplay}>
+        {
+          this.state.displayList &&
+          <div id='recent-song-list'>
+            {songs.map(song =>
+              <div key={song.id} class='recent-song-info'>
+                <img src={song.art} alt={`Album Title: ${song.album}`}/>
+                <p>
+                  <b>Song Title: {song.title}</b><br />
+                  Artist: {song.artist}
+                </p>
+              </div>
+            )}
+          </div>
+        }
+         <p id='recently-played-text'>Recently Played Songs</p>

```

Comment:
nit, but maybe we could change this to `Recent songs` to make it shorter? 

It wraps to a new line on my laptop screen

<img width="611" alt="Screen Shot 2020-02-14 at 10 11 17 AM" src="https://user-images.githubusercontent.com/25715018/74498198-60303880-4f12-11ea-9210-5b95c62888db.png">

And when testing with responsive mode, each word has its own line

<img width="532" alt="Screen Shot 2020-02-14 at 10 10 48 AM" src="https://user-images.githubusercontent.com/25715018/74498224-7a6a1680-4f12-11ea-8e88-a60be9fe05b8.png">